### PR TITLE
Definitive parts for Win 8.1, using the real names from the PDBs

### PR DESCRIPTION
**Pull request by memoarfaa**
update theme part and state map for DWMWINDOW WIN81

### CODE REVIEW

File: libmsstyle/VisualStylePartMap.cs [hunk 1]
```diff
@@ -497,76 +497,77 @@ public class VisualStyleParts
...
-            new VisualStylePartEntry(43, "TOOLFRAMELEFT", VisualStyleStates.STATES_WINDOW_FRAME_GEN ),
-            new VisualStylePartEntry(44, "TOOLFRAMERIGHT", VisualStyleStates.STATES_WINDOW_FRAME_GEN ),
-            new VisualStylePartEntry(45, "TOOLFRAMETOP", VisualStyleStates.STATES_WINDOW_FRAME_GEN ),
+            new VisualStylePartEntry(1, "BOTTOMFRAME", VisualStyleStates.STATES_WINDOW_FRAME_GEN ),
+            new VisualStylePartEntry(2, "BOTTOMSHADOW", VisualStyleStates.STATES_WINDOW_FRAME_GEN ),
+            new VisualStylePartEntry(3, "BUTTONACTIVECAPTION", VisualStyleStates.STATES_WINDOW_BTN_AND_THUMB ),
+            new VisualStylePartEntry(4, "BUTTONINACTIVECAPTION", VisualStyleStates.STATES_WINDOW_BTN_AND_THUMB ),
+            new VisualStylePartEntry(5, "BUTTONACTIVECAPTIONEND", VisualStyleStates.STATES_WINDOW_BTN_AND_THUMB ),
+            new VisualStylePartEntry(6, "BUTTONINACTIVECAPTIONEND", VisualStyleStates.STATES_WINDOW_BTN_AND_THUMB ),
+            new VisualStylePartEntry(7, "BUTTONACTIVECLOS", VisualStyleStates.STATES_WINDOW_BTN_AND_THUMB ),
```
Comment (nptr) [inline libmsstyle/VisualStylePartMap.cs:510]: The trailing "E" got lost here i think.

Comment (nptr) [reply libmsstyle/VisualStylePartMap.cs:510]: Thanks a lot! I'll test and merge it once I boot up my Windows machine again.

**Comment by nptr** [comment]
Thanks a lot! I'll test and merge it once I boot up my Windows machine again.

**Comment by nptr** [comment]
I fixed the typo and rebased your changes with commit 7a4ff2238018226aa08e47c8a250584e95b839fe.
Version [2.0.0.4](https://github.com/nptr/msstyleEditor/releases/tag/2.0.0.4) is the first version to include them.

**Comment by memoarfaa** [comment]
Ok very thanks

**Pull request closed without merging**